docs/basics/supported_systems.md

```diff
@@ -0,0 +1,22 @@
+---
+icon: octicons/cpu-24
+search:
+    boost: 4
+---
+# :octicons-cpu-24: Supported Systems
+
+## :fontawesome-solid-computer: Operating Systems
+Currently, **libdebug** only supports the :simple-linux: GNU/Linux Operating System.
+
+## :fontawesome-solid-microchip: Architectures
+
+
+| Architecture                                                                 | Alias                     | Support                           |
+| :--------------------------------------------------------------------------- | :-----------------------: | :------------------------------: |
```

Comment:
is really everything supported stable? It will be the first release for more than half of the archs described

Reply:
[9e99adc](https://github.com/libdebug/libdebug/pull/129/commits/9e99adc7c4f740849576a6ec29d1c746c4e2a5ef)